testutil/test_helper.go

```diff
@@ -91,7 +95,16 @@ func NewGenericTestHelper(
 		clusterDomain:  clusterDomain,
 		externalIssuer: externalIssuer,
 		uninstall:      uninstall,
+		httpClient:     httpClient,
 	}
+
+	kubernetesHelper, err := NewKubernetesHelper(k8sContext, h.RetryFor)
```

Comment:
As the `kubernetesHelper` is public, and so is `NewKubernetesHelper()`, why can't we just take it as a argument and assign it directly, rather than using this function directly. Any upsides to this?

Reply:
Done!